### PR TITLE
Docs: Mosviz Pan/Zoom and Spectral Regions

### PR DESCRIPTION
**Pull request by dtn5ah**
Some mosviz documentation for pan/zoom and spectral regions. For the most part, I just linked the specviz docs as they're basically the same with a few caveats:
1. I added special section talking about the linked 1D and 2D spectral viewers
2. I am restricting the spectral region to the 1D spectral viewer, ignoring the 2D spectral viewer and image viewer. Playing around with them, I don't think those features are finalized and I struggled to get consistent behavior to document. Also because we are considering creating a dedicated 2D spectral viewer anyways, I think we shouldn't invest too heavily into docs of the 2D spectral viewer just yet

### CODE REVIEW

**Comment by dtn5ah** [comment]
For some reason, codecov is showing coverage decreases on four files that I don't touch with this PR. Any reason why this could be failing

File: docs/mosviz/displayspectra.rst [hunk 1]
```diff
@@ -7,12 +7,19 @@ Coming soon
 Pan/Zoom
 ========
 
-Coming soon
+For an overview on general pan/zoom mechanics, please see :ref:`pan-zoom`
```
Comment (PatrickOgle) [inline docs/mosviz/displayspectra.rst:10]:
```suggestion
For an overview on general pan/zoom functionality, please see :ref:`pan-zoom`
```

**Review by PatrickOgle — APPROVED** [review]
Suggested one change for your consideration, otherwise approve.

**Review by javerbukh — APPROVED** [review]
Looks good! I'll give you the opportunity to implement Patrick's suggestion (if you agree with it) before I merge.

**Comment by javerbukh** [comment]
Merging.

**Pull request merged**